config/engines.yml

```diff
@@ -12,9 +12,9 @@
 brakeman:
   image: codeclimate/codeclimate-brakeman
   description: Static analysis tool which checks Ruby on Rails applications for security vulnerabilities.
-  community: true
+  community: false
   enable_regexps:
-    - \.rb$
+    - ^script\/rails$
```

Comment:
We probably want to add back `upgrade_languages` here: the upgrade languages are an and condition with the `enable_regexps`: classic repos need to trigger both to get the engine.